uaclient/util.py

```diff
@@ -477,3 +477,10 @@ def print_package_list(
         )
     )
     print("")
+
+
+def get_user_or_root_log_path() -> str:
+    if we_are_currently_root():
+        return "/var/log/ubuntu-advantage.log"
```

Comment:
We can extract that information directly through the `UAConfig` object. Even though users shouldn't change that option, it is still safer to consume that from an `UAConfig` objectt.

Reply:
Thank you for this suggestion! I was wondering if there was somewhere I could consume this info from, so thank you for pointing me in the right direction